src/i18n/strings/en_EN.json

```diff
@@ -3137,11 +3136,12 @@
     "Really reset verification keys?": "Really reset verification keys?",
     "Skip verification for now": "Skip verification for now",
     "Failed to send email": "Failed to send email",
-    "Changing your password will reset any end-to-end encryption keys on all of your sessions, making encrypted chat history unreadable. Set up Key Backup or export your room keys from another session before resetting your password.": "Changing your password will reset any end-to-end encryption keys on all of your sessions, making encrypted chat history unreadable. Set up Key Backup or export your room keys from another session before resetting your password.",
+    "Signing out all sessions will reset the end-to-end encryption keys making encrypted chat history unreadable. Set up Key Backup or export your room keys from another session before resetting your password.": "Signing out all sessions will reset the end-to-end encryption keys making encrypted chat history unreadable. Set up Key Backup or export your room keys from another session before resetting your password.",
```

Comment:
And of course this needs changing too if we go [with my suggestion above](https://github.com/matrix-org/matrix-react-sdk/pull/8259/files#r846011928).